docs/how-to/explore/find-any-resource.md

```diff
@@ -0,0 +1,77 @@
+---
+sidebar_label: Find any resource
+sidebar_custom_props:
+  tags: [AWS, GCP, DigitalOcean, Kubernetes]
+---
+
+# How to quickly find any resource
+
+## Introduction
+
+Resoto maintains an asset inventory of your infrastructure metadata. It is equipped with a full-text search index that can be leveraged to find any resource, no matter which cloud provider, account, region or service the resource belongs to.
```

Comment:
didn't we edit this paragraph already elsewhere?

Reply:
This was the result of a bad rebase.